distributed/scheduler.py

```diff
@@ -4206,7 +4215,7 @@ async def start(self):
         for k, v in self.services.items():
             logger.info("%11s at: %25s", k, "%s:%d" % (listen_ip, v.port))
 
-        self.loop.add_callback(self.reevaluate_occupancy)
+        self.loop.add_callback(copy_context().run, self.reevaluate_occupancy)
```

Comment:
we should not be needing a copy_context at this point. the reevaluate occupancy is not mutating any TaskState

Reply:
removed